src/index.ts

```diff
@@ -1,3 +1,3 @@
 export { compile } from './compile';
 export { render } from './render';
-export { convert as default } from './convert';
```

Comment:
Why did you change the `convert()` function from default export to named export?

Reply:
This is because import using `require()` is complicated.

If you are using ES Modules ( TypeScript ), you can import it as follows:

```js
// ES Modules
import convert from 'metalsmith-pug-extra';
import { compile, render } from 'metalsmith-pug-extra';
```

But if you are using CommonJS, you need to import it like this:

```js
// CommonJS
const { default: convert } = require('metalsmith-pug-extra');
const { compile, render } = require('metalsmith-pug-extra');
```

The need for the `default` keyword is not intuitive.

---

Ideally, for code that uses CommonJS, it is best to import as follows:

```js
// CommonJS
const convert = require('metalsmith-pug-extra');
const { compile, render } = require('metalsmith-pug-extra');
```

However, such build results could not be expressed in TypeScript.
For this reason, I prioritized being able to write similar code in ES Modules and CommonJS.
That is, it looks like this:

```js
// ES Modules
import { convert } from 'metalsmith-pug-extra';
import { compile, render } from 'metalsmith-pug-extra';

// CommonJS
const { convert } = require('metalsmith-pug-extra');
const { compile, render } = require('metalsmith-pug-extra');
```